packages/foundation/src/elements/breadcrumb.css

```diff
@@ -0,0 +1,53 @@
+.hap-breadcrumb {
+  font-family: var(--hap-typography-font-family-body);
+  font-size: var(--hap-typography-font-size-bodytext-s);
+  font-weight: var(--hap-typography-font-weight-medium);
+  letter-spacing: var(--hap-typography-letter-spacing-lg);
+  line-height: var(--hap-typography-line-height-bodytext-s-singleline);
+  gap: var(--hap-spacing-xs);
+
+  align-items: center;
+
+  color: light-dark(
+    var(--hap-color-text-tertiary-on-light),
+    var(--hap-color-text-tertiary-on-dark)
+  );
+  background-color: light-dark(
+    var(--hap-color-text-tertiary-on-dark),
+    var(--hap-color-text-tertiary-on-light)
+  );
+
+  .hap-navList {
```

Comment:
Die Struktur passt, das Naming ist noch nicht optimal:
- Klassennamen sollten komplett in kebab-case geschrieben sein
- anstatt "nav***" würde ich "breadcrumb-***" bevorzugen. Dann sieht man am naming eindeutig, dass das alles zusammen gehört.